src/Cache/ArrayResult.php

```diff
@@ -107,7 +107,7 @@ public function __serialize(): array
         return [$this->columnNames, $this->rows];
     }
 
-    /** @param mixed[] $data */
+    /** @param array{0: list<string>, 1: list<list<mixed>>} $data */
```

Comment:
```suggestion
    /** @param array{list<string>, list<list<mixed>>} $data */
```

The keys are not necessary. Let's match the style we used in line 104.

Reply:
Fixed at 604bca208